Packs/Absolute/Integrations/Absolute/Absolute.py

```diff
@@ -893,6 +1063,86 @@ def get_device_location_command(args, client) -> CommandResults:
             readable_output=f"No device locations found in {INTEGRATION} for the given filters: {args}")
 
 
+''' EVENT COLLECTOR '''
+
+
+def fetch_events(client: ClientV3, fetch_limit: int, last_run: Dict[str, Any]) -> tuple[List[Dict[str, Any]], Dict[str, Any]]:
+    next_page_token = last_run.get('next_page_token', '')
+    last_run_latest_events_id = last_run.get('latest_events_id', [])
+    last_end_date = last_run.get('end_date', '')
+    last_start_date = last_run.get('start_date', '')
+    end_date = datetime.utcnow()
+
+    # start_date = datetime.strptime(last_start_date, "%Y-%m-%d %H:%M:%S") if next_page_token else (
+    #     datetime.strptime(last_end_date, "%Y-%m-%d %H:%M:%S") if last_end_date else end_date - timedelta(minutes=1))
+
+    # TODO: remove this temp start_date
+    start_date = datetime.strptime(last_start_date, "%Y-%m-%d %H:%M:%S") if next_page_token else (
+        datetime.strptime(last_end_date, "%Y-%m-%d %H:%M:%S") if last_end_date else end_date - timedelta(minutes=7))
+    # TODO: remove this temp start_date
+
+    demisto.debug(f'Starting new fetch: {fetch_limit=}, {start_date=}, {end_date=}, {next_page_token=}, {last_run=}')
+    all_events, next_page_token = client.fetch_events_with_pagination(fetch_limit, next_page_token, start_date, end_date)
+    if not all_events:
+        return [], {
+            'next_page_token': '',
+            'start_date': start_date.strftime("%Y-%m-%d %H:%M:%S"),
+            'end_date': end_date.strftime("%Y-%m-%d %H:%M:%S"),
+            'latest_events_id': []}
+
+    events, latest_events_id = handle_duplication_and_add_time_field_to_events_and_get_latest_events(all_events,
```

Comment:
if all_events is empty, the handle_duplication_and_add_time_field_to_events_and_get_latest_events method should return empty list of events and empty array of latest_events_id. If so, we can remove the if condition and leave just the return statement at the end of the method.